pedpy/methods/flow_calculator.py

```diff
@@ -72,6 +72,8 @@ def compute_n_t(
 
     # frame number is the index
     n_t[TIME_COL] = n_t.index / traj_data.frame_rate
+    n_t = n_t.reset_index()
+    n_t.columns = [FRAME_COL, "cumulative_pedestrians", TIME_COL]
```

Comment:
Please use the constants for the column names instead of strings:

```suggestion
    n_t.columns = [FRAME_COL, CUMULATED_COL, TIME_COL]
```

CUMULATED_COL may need to be imported on top of the file.